### PR TITLE
PHP7.2 count fixes

### PR DESCRIPTION
**Pull request by snipe**
This *should* fix some of the count errors we're running into when the object being counted is null. I've re-enabled travis tests for 7.2 for now so we can track down places I missed.

@dmeltzer @uberbrady if either/both of you could eyeball this, seeing as it's 3:22AM, I might have donked something up :)

### CODE REVIEW

**Comment by snipe** [comment]
This is still failing on travis for 7.2, but for the same weird updated_at reasons that it usually fails for. 

I'm not really sure why we're getting a off-by-one-second Carbon parse, TBH... It doesn't really make sense.

**Comment by dmeltzer** [comment]
Looks good to me.  I wonder if any of these placed would benefit from a withCount()/counting on the db side instead of ->count on the fetched collection?  Not sure if theres a performance difference.

**Comment by snipe** [comment]
Testing linking back from discourse. Ignore this :)
http://discourse.snipeitapp.com/t/this-is-a-test-post-in-bug-reports/17/2

File: app/Http/Controllers/Api/CustomFieldsController.php [hunk 1]
```diff
@@ -26,9 +26,7 @@ public function index()
     {
         $this->authorize('index', CustomFields::class);
         $fields = CustomField::get();
-
-        $total = count($fields);
-        return (new CustomFieldsTransformer)->transformCustomFields($fields, $total);
+        return (new CustomFieldsTransformer)->transformCustomFields($fields, $fields->count());
```
Comment (snipe) [inline app/Http/Controllers/Api/CustomFieldsController.php]: Testing linking back from discourse via commit comment. Ignore this :)
http://discourse.snipeitapp.com/t/this-is-a-test-post-in-bug-reports/17/2

**Comment by snipe** [comment]
This pull request has been mentioned on **Snipe-IT Discussion**. There might be relevant details there:

http://discourse.snipeitapp.com/t/testing-the-gh-linkback-plugin/28/1

**Pull request merged**